### ISSUE TITLE
BERT 输出问题

### ISSUE THREAD

**Issue by Ruri-94** [PaddlePaddle/PaddleNLP#1711]
相同的输入 每次输出的结果都是不一样的是什么问题？

import paddle
from paddlenlp.transformers import *

tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
model = BertModel.from_pretrained('bert-base-uncased')
inputs = tokenizer("Welcome to use PaddlePaddle and PaddleNLP!")
inputs = {k:paddle.to_tensor([v]) for (k, v) in inputs.items()}

output, _ = model(**inputs,output_hidden_states=True)
print(output[-1])

**Comment by wj-Mcat** [comment]
There are a little bit of premise: 
* If the model is in training mode, the dropout will be disabled. you can refer to: [dropout](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/functional/common.py#L743)

```python
```

*

**Comment by ZHUI** [comment]
可以设置 `model.eval()` 模型中的 dropout 会有一定随机性。

**Comment by wj-Mcat** [comment]
There are a little bit of premises: 

If the model is in training mode, the dropout will be disabled. you can refer to: [dropout](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/functional/common.py#L743) which the default mode is training.

the unit test code will pass:
```python
from paddle.nn import Dropout
tensor = paddle.randn((2,3))
dropout = Dropout(p=0.5)
dropout.training = False

first_tensor = dropout(tensor)
second_tensor = dropout(tensor)
assert paddle.all(paddle.equal(first_tensor, second_tensor)).detach().item()
```

So you can set model eval mode to disable dropout layer.  

```python
import paddle
from paddlenlp.transformers import *

tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
model = BertModel.from_pretrained('bert-base-uncased')
model.eval()
inputs = tokenizer("Welcome to use PaddlePaddle and PaddleNLP!")
inputs = {k:paddle.to_tensor([v]) for (k, v) in inputs.items()}
output, _ = model(**inputs,output_hidden_states=True)
print(output[-1])
```

so you will get the same output when you run it.  Hope that can help you understand it.
